backend/src/main/java/app/domain/round/RoundPlayerService.java

```diff
@@ -72,10 +77,24 @@ void setNextPlayer() {
         }
     }
 
-    void fold() {
-        RoundPlayer player = roundPlayers.pollFirst();
-        player.fold();
-        roundPlayers.addLast(player);
+    void putProperPlayerOnTop() {
+        RoundPlayer player = roundPlayers.getFirst();
+        if (!player.getPlayerPosition().equals(Position.SMALL_BLIND)) {
```

Comment:
Nie mam tego odpalonego. A co jak player na Small blindzie będzie sfoldowany albo na all inie? Wtedy będzie miał turę chociaż powinien mieć gość po nim

Reply:
jak bedzie na small blindzie to wykona sie getFirstPlayerInGame(), które sprawdzi czy jest w grze a jesli nie to wezmie następnego